MAINTAINERS.md

```diff
@@ -1,5 +1,97 @@
 # Maintainers
 
+OPI maintainers do the work of keeping the project running.
+
+Maintainers do the following things in support of OPI projects:
+
+* Writing code for new features
+* Writing code to fix bugs
+* Opening bug reports
+* Triaging bug reports
+* Closing bug reports
+* Reviewing pull requests
+* Merging pull requests from contributors
+* Resolving conflict found on pull request reviews
+* Proactively dealing with technical debt
+* Supporting documentation
+* Guiding the projects strategic direction
+
+In other words, maintainers are involved in day-to-day operations of the
+project.
+
+## OPI Maintainer Prerequisites
+
+You should understand the [contributing](https://github.com/opiproject/opi/blob/main/CONTRIBUTING.md)
+guidelines if you want to become a maintainer.
+
+## OPI Maintainer Grant/Revocation Policy
+
+An OPI maintainer is a participant in OPI who has the ability to merge pull
+requests directly into main branch on OPI repositories. This ability grants
+the maintainers the ability to affect the progress of the OPI project in a way
+non-maintainers cannot. Being an OPI maintainer represents a significant level
+of trust to the broader project. Maintainer rights are not granted lightly,
+and will be reviewed as necessary to ensure that trust has not been violated.
+
+### Granting Maintainer Rights
+
+Granting maintainer rights will be considered when a candidate has demonstrated
+the following in their interation with the project:
+
+* Contribution of significant new features to OPI.
+  * Submissions are free of obvious defects.
+  * Submissions do not require many revisions.
+* Consistent and meaningful code reviews of other people's pull requests.
+* Helping others in the community via channels such as the mailing list,
+  GitHub. Issues and PRs, and the Slack channels
+* The individual should have plans for sustained contribution to the project.
+
+The process to grant someone the maintainer role is simple:
+
+* An existing maintainer nominates someone by sending a message to the
+  maintainers slack channel.
+* Existing maintainers discuss the idea of giving the candidate maintainer
+  rights.
+* After discussion has ended, existing maintainers will vote on approving the
```

Comment:
what if TSC/Board disagree with the vote ?

Reply:
Two things:
1. The TSC needs to evolve to become maintainers and people who write code.
2. We cannot and should not have a model where TSC appoints maintainers.

Maintainers do grunt work and day-to-day coding and maintenance of the code bases. They need to self-select because it's a job, not a privilege.

Reply:
I like how you phrased it and support it